src/main/java/io/odpf/kafkaclient/producer/Producer.java

```diff
@@ -1,6 +1,51 @@
 package io.odpf.kafkaclient.producer;
 
 import io.odpf.kafkaclient.Client;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.concurrent.Future;
 
 public class Producer extends Client {
+
+    private final KafkaProducer<String, String> kafkaProducer;
+    private final BufferedReader bufferedReader;
+
+    public Producer(KafkaProducer<String, String> kafkaProducer) {
+        this.kafkaProducer = kafkaProducer;
+        bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+    }
+
+    public Future<RecordMetadata> writeEvent(String... eventData) {
```

Comment:
Why public?

Reply:
Changed access level to default,(in order to be accessed by ProducerTest), in a later commit